RecoTracker/MkFitCMS/src/MkStdSeqs.cc

```diff
@@ -0,0 +1,586 @@
+#include "RecoTracker/MkFitCMS/interface/MkStdSeqs.h"
+
+#include "RecoTracker/MkFitCore/interface/HitStructures.h"
+#include "RecoTracker/MkFitCore/interface/IterationConfig.h"
+
+#include "RecoTracker/MkFitCore/interface/binnor.h"
+
+#include "tbb/parallel_for.h"
+
+namespace mkfit {
+
+  namespace StdSeq {
+
+    //=========================================================================
+    // Hit processing
+    //=========================================================================
+
+    void LoadDeads(EventOfHits &eoh, const std::vector<DeadVec> &deadvectors) {
+      for (size_t il = 0; il < deadvectors.size(); il++) {
+        eoh.SuckInDeads(int(il), deadvectors[il]);
+      }
+    }
+
+    // Loading hits in CMSSW from two "large multi-layer vectors".
+    // orig_hitvectors[0] - pixels,
+    // orig_hitvectors[1] - strips.
+
+    void Cmssw_LoadHits_Begin(EventOfHits &eoh, const std::vector<const HitVec *> &orig_hitvectors) {
+      eoh.Reset();
+
+      for (auto &&l : eoh.m_layers_of_hits) {
+        // This slightly sux.
+        l.BeginRegistrationOfHits(*orig_hitvectors[l.is_pix_lyr() ? 0 : 1]);
+      }
+    }
+
+    // Loop with LayerOfHits::RegisterHit(int idx) - it takes Hit out of original HitVec to
+    // extract phi, r/z, and calculate qphifines
+    //
+    // Something like what is done in MkFitInputConverter::convertHits
+    //
+    // Problem is I don't know layers for each large-vector;
+    // Also, layer is calculated for each detset when looping over the HitCollection
+
+    void Cmssw_LoadHits_End(EventOfHits &eoh) {
+      for (auto &&l : eoh.m_layers_of_hits) {
+        l.EndRegistrationOfHits(false);
+      }
+    }
+
+    //=========================================================================
+    // Hit-index mapping / remapping
+    //=========================================================================
+
+    void Cmssw_Map_TrackHitIndices(const EventOfHits &eoh, TrackVec &seeds) {
+      for (auto &&track : seeds) {
+        for (int i = 0; i < track.nTotalHits(); ++i) {
+          int hitidx = track.getHitIdx(i);
```

Comment:
const here and below?

Reply:
is this a comment for `auto &&track : seeds` or for `int hitidx = track.getHitIdx(i)` ?

Reply:
Addressed in Round 1 follow-up PR.
trackreco/cmssw#75 https://github.com/trackreco/cmssw/pull/75/commits/e9834b4a33dab0a34078a4861b825f0e1ebdb64d

Reply:
it was a comment for the ints.